app/views/admin/loans/timeline/_step.html.slim

```diff
@@ -2,7 +2,7 @@
 - completed = entry.completed? ? "completed" : ""
 
 / Note that we have to repeat these classes on all td tags because we don't always have access to the tr tag
-- classes = "#{completed} #{draft}"
+- classes = "#{completed} #{draft} project-step"
```

Comment:
Added so that hovering over any part of the step will show related step relationships.